style.css

```diff
@@ -1 +1,156 @@
-/*  Add your styling here */
+.logo{
+    background-color: hotpink;
+    height: 50px;
+}
+
+
+nav {
+    display: flex;
+    flex-direction: row;
+    background-color: gray;
+}
+
+.navbar{
+    display: flex;
+    flex-direction: row;
+    background-color: grey;
+    
+}
+
+ul{
+    list-style: none;
+}
+
+.pic1{
+    background-color: indigo;
+    width: 80px;
+}
+
+.pic2{
+    background-color:yellow;
+    width: 80px;
+}
+.pic3{
+    background-color: lightblue;
+    width: 80px;
```

Comment:
instead of having a fixed width for your images, instead use relative sizes like percenatges so that the elements with this class can then scale responsively.